src/ng2-auto-complete.directive.ts

```diff
@@ -46,6 +47,7 @@ export class Ng2AutoCompleteDirective implements OnInit {
   inputEl: HTMLInputElement;  // input tag
 
   constructor(private resolver: ComponentFactoryResolver,
+              private renderer: Renderer,
               public  viewContainerRef: ViewContainerRef) {
     this.el = this.viewContainerRef.element.nativeElement;
   }
```

Comment:
i blame webstorm!

i would rather something like 
```
constructor(
  private A,
  private B) {}
```
If this is fine I'll change them all to match